rag/ai-studio.yaml

```diff
@@ -0,0 +1,20 @@
+application:
+  type: language
+  name: rag-demo
+  description: This is a RAG demo application. 
+  containers:
+    - name: rag-demo-service
+      contextdir: model_services
+      containerfile: base/Containerfile
+      model-service: true
+      backend: 
+        - llama
+      arch:
+        - arm64
+        - amd64
```

Comment:
I think we should add a flag so we know that it supports rag, and maybe the type if there could be alternatives (e.g this works with BAAI/bge-base-en-v1.5).

Reply:
sure, what would that look like:

```
backend:
   - llama
rag:
   - True
```

What would that be used for?  Right now this example uses an in-memory vectordatabase in the same container. However, I would like to extend this example (in a follow up PR). Where the vectordatabase is a stand alone container. How would that impact the `ai-studio.yaml` file?

Reply:
In that case we should add a completely new container to the list. We can do it later when you push the follow up PR